api/url_mapping.go

```diff
@@ -194,4 +195,16 @@ func MapUrl(app *fiber.App, handlers ...fiber.Handler) {
 	stacksNodesGroup.Get("/:name/status", websocket.New(shared.Status))
 	stacksNodesGroup.Get("/:name/metrics", websocket.New(shared.Metrics))
 
+	aptosGroup := v1.Group("aptos")
+	aptosNodesGroup := aptosGroup.Group("nodes")
+	aptosNodesGroup.Post("/", middleware.IsDuplicated, aptos.Create)
+	aptosNodesGroup.Get("/:name", stacks.ValidateNodeExist, aptos.Get)
```

Comment:
use replace stacks with aptos
```suggestion
	aptosNodesGroup.Get("/:name", aptos.ValidateNodeExist, aptos.Get)
```